Inferences/Induction.cpp

```diff
@@ -190,6 +200,20 @@ Formula* InductionContext::getFormulaWithSquashedSkolems(const std::vector<TermL
   return res;
 }
 
+bool InductionContext::getFreeVariable(unsigned& res) const {
+  // Note: we assume there is just one free variable in the literals of _cls
+  for (const auto& kv : _cls) {
+    for (const auto& lit : kv.second) {
+      FormulaVarIterator fvi(lit);
```

Comment:
How about using the less heavy `VariableIterator`? Then we don't need the include above either.

Reply:
Fair point, done.